mdb-realm/README.md

```diff
@@ -2,40 +2,38 @@
 
 This action exposes an API to create an Atlas cluster and then import previously exported MDB Realm applications.
 
-## deployApps
+## deploy
```

Comment:
Should we rename it to `deployCluster`?

Reply:
I thought `deploy` fits better the `cleanup` action (better symmetry). But it's not a strongly held belief - if you find `deployCluster` more descriptive and useful, I'd be fine with renaming it.